Update automatic-vm-guest-patching.md to indicate availability timelines for Windows Server 2025 Azure Edition (#93)
MicrosoftDocs/azure-compute-docs: articles/virtual-machines/automatic-vm-guest-patching.md

> Automatic VM guest patching, on-demand patch assessment and on-demand patch installation are supported only on VMs created from images with the exact combination of publisher, offer and sku from the below supported OS images list. Custom images or any other publisher, offer, sku combinations aren't supported. More images are added periodically. Don't see your SKU in the list? Request support by filing out [Image Support Request](https://forms.microsoft.com/r/6vfSgT0mFx).

### Supported Windows Images (Hotpatchable)

| Publisher               | OS Offer      |  Sku               |
|-------------------------|---------------|--------------------|
| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-core |
| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-core-smalldisk |
| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-hotpatch |
| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-hotpatch-smalldisk |

> [!IMPORTANT] 

> > Support for the following **Windows Server 2025** images is currently being rolled out. Regions where support doesn't exist yet may return a HTTP 400 client error indicating an unsupported status in the interim. You may successfully use one of the regions with support like UK South, East Asia or West Central US instead. This advisory will be removed when support is available everywhere.

Reviewer: ```suggestion
> Not all regions currently support **Windows Server 2025** images. Regions where support doesn't exist yet may return a HTTP 400 client error, indicating an unsupported status. Instead, use another region that does provide support, such as, UK South, East Asia, or West Central US.
```

Author: @ju-shim We can take that change but will need to be more specific then --
Not all regions currently support **Windows Server 2025 Azure Edition** images. Regions where support doesn't exist yet may return a HTTP 400 client error, indicating an unsupported status. Instead, use another region that does provide support, such as, UK South, East Asia, or West Central US.

Reviewer: @kjohn-msft - Yeah feel free to make that change, it's perfect. Let me know when you are done, and I can sign this off to go live on Nov 19th at 10am (PST). Per my email thread response, we are in a companywide publishing freeze for Ignite, so all content signed off right now will go love that morning and can't go sooner. Thank you! :)

Reviewer: I updated my suggested so it's easy to add as a commit.
